composer.json

```diff
@@ -32,7 +32,8 @@
     "deliciousbrains/wp-background-processing": "1.0.2",
     "lifterlms/lifterlms-blocks": "2.0.0-beta.6",
     "lifterlms/lifterlms-rest": "1.0.0-beta.19",
-    "woocommerce/action-scheduler": "3.2.0"
+    "woocommerce/action-scheduler": "3.2.0",
+    "lifterlms/lifterlms-helper": "9999999-dev"
```

Comment:
is this on purpose?

Reply:
Yes, just for testing, that's equivalent to "dev-trunk" since there wasn't a composer installable release available (yet). Updating with the proper version now that it's released